### PR TITLE
🐌

### PR DESCRIPTION
**Pull request by robherley**
i made a ui

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      }
+    </style>
+  </head>
+  <body>
+    <form method="post">
+      <h1>🐌</h1>
+      <input
+        name="url"
+        type="url"
+        placeholder="https://tiny.coffee"
```
Comment (benwaffle) [inline static/index.html]: pls remove

Comment (robherley) [reply static/index.html]: what do you want in placeholder

Comment (benwaffle) [reply static/index.html]: hmm

**Comment by robherley** [comment]
i updated it

File: static/index.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      button:hover,
+      button:focus {
+        filter: brightness(120%);
+        cursor: pointer;
+      }
+    </style>
+  </head>
+  <body>
+    <form method="post">
+      <h1>🐌</h1>
```
Comment (benwaffle) [inline static/index.html:85]: that's a snail

**Pull request merged**